core/templates/pages/donate-page/donate-page.component.ts

```diff
@@ -154,4 +202,15 @@ export class DonatePageComponent implements OnInit {
       windowClass: 'donation-box-modal',
     });
   }
+
+  nextTile(clickedVal: number): void {
+    let learnerTiles = document.getElementsByClassName('tile-' + clickedVal);
```

Comment:
Should we consider using template references for the tiles instead -- this might be a more Angular way of referencing HTML elements?

See https://stackoverflow.com/a/47616844

Reply:
Done, good point. Used template refs instead of document.getElementsByClassName().